spec/restaurant_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'restaurant'
+
+describe Restaurant do
+  let(:customer) {double (:customer)}
+  let(:menuClass){double :menuClass, :new => menu}
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Space missing to the left of {.<br>Space missing inside {.<br>Space missing inside }.